### DISCUSSION TITLE
Mainline and/or Linux-next Zen Kernel

### DISCUSSION THREAD

**Discussion by The-Last-Soldier** [zen-kernel/zen-kernel#320] [Ideas]
I think it would be a great idea to have the **_Zen Kernel_** to also have a **_mainline_** version in the aur then we could have the lastest performance improvements that that the **_mainline_** offers combined with the **_zen kernel_**. for the extra performance improvement they both bring combined I think would make a great kernel, especially for gamers. (Please consider this cause it is something I'd like to see/use)

**Comment by damentz** [comment]
How is this different than the stock Arch Linux kernel?

**Reply by heftig** [reply]
I think they want us to maintain the patchset for `rc` releases as well.

**Reply by The-Last-Soldier** [reply]
It's newer mainline is at version 6.5-rc3 while stable is at 6.4.7 (I'm currently running the one even newer than mainline linux-next)

**Reply by damentz** [reply]
@The-Last-Soldier You're asking @heftig to run the rc kernels for you to make sure they work, weeks before patches we depend on are ready.  Without asking, the answer is probably never, unless you or someone sponsored @heftig to do that.
